.eslintignore

```diff
@@ -1,6 +1,3 @@
-*
-!**/
-!**/lwc/**/*.js
 node_modules
```

Comment:
Updated so `eslint --max-warnings 0 "**/*.js"` doesn't give any warnings for ignored files.